examples/step-44/doc/intro.dox

```diff
@@ -109,29 +109,29 @@ such that
 		\qquad \text{and} \qquad
 	\dfrac{1}{2}[\mathbf{A} - \mathbf{A}^T] = \mathcal{W}:\mathbf{A} \, .
 @f]
-The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{S}$.
+The fourth-order <code>SymmetricTensor</code> returned by identity_tensor() is $\mathcal{I}$.
 
 
 <h3>Kinematics</h3>
 
 Let the time domain be denoted $\mathbb{T} = [0,T_{\textrm{end}}]$, where $t \in \mathbb{T}$ and $T_{\textrm{end}}$ is the total problem duration.
 Consider a continuum body that occupies the reference configuration $\Omega_0$ at time $t=0$.
-Particles in the reference configuration are identified by the position vector $\mathbf{X}$.
+%Particles in the reference configuration are identified by the position vector $\mathbf{X}$.
 The configuration of the body at a later time $t>0$ is termed the current configuration, denoted $\Omega$, with particles identified by the vector $\mathbf{x}$.
 The nonlinear map between the reference and current configurations, denoted $\boldsymbol{\varphi}$, acts as follows:
 @f[
 	\mathbf{x} = \boldsymbol{\varphi}(\mathbf{X},t) \, .
 @f]
 The material description of the displacement of a particle is defined by
 @f[
-	\mathbf{U}(\mathbf{X},t) = \mathbf{x}(\mathbf{X},t) - \mathbf{X} \, .
+	\mathbf{U}(\mathbf{X},t) = \boldsymbol{\varphi}(\mathbf{X},t) - \mathbf{X} \, .
```

Comment:
Although not incorrect, it is common in solid mechanics textbooks to just write "x = X + u". That is reflected here, so please change it back.

Reply:
Sure.

Reply:
Thanks!